plugin/xcodebuild.vim

```diff
@@ -80,6 +80,14 @@ function! s:sdk()
   endif
 endfunction
 
+function! s:runner()
+  if exists('g:xcodebuild_runner')
+    return g:xcodebuild_runner
+  else
+    return '!'
```

Comment:
~~The `!` feels like a magic \~\~number\~\~ character here. Maybe extract and name this to something like `s:xcodebuild_default_background_runner`? You can pull the `let s:xcodebuild_default_background_runner = '!'` to the top of the script if you go that route.~~

Commented more completely above with configuration suggestion.